clustering/ejb/infinispan/src/main/java/org/wildfly/clustering/ejb/infinispan/InfinispanBeanManager.java

```diff
@@ -407,13 +417,17 @@ public T acquire() {
 
         @Override
         public boolean release() {
+            schedule(bean);
```

Comment:
I don't think it makes sense to schedule expiration/passivation on release.  The last access time isn't updated until close(), and we already schedule for expiration/passivation then.

Reply:
that's a great catch, I updated the PR with this